README.md

```diff
@@ -60,4 +60,18 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 ### Tests
 There are 3 types of tests that have been added - Unit Tests , Integration Tests and EndToEnd Tests.
 They can be run by the following command : python -m pytest
-For the end to end tests , install the latest version of Firefox browser and geckodriver.
\ No newline at end of file
+For the end to end tests , install the latest version of Firefox browser and geckodriver.
+
+### Running the Tests via docker
+
+### Build the docker image for tests
+docker build --target test --tag my-test-image .
+
+### Running the Unit Tests via docker
+docker run my-test-image tests/unit
+
+### Running the Integration Tests via docker
+docker run my-test-image tests/integration
+
+### Running the E2E Tests via docker , the environment variables need to be passed
+docker run --env apiKey --env apiToken --env boardId --env TODO_LIST_ID --env DOING_LIST_ID --env DONE_LIST_ID --env FLASK_APP --env FLASK_ENV my-test-image tests/endtoend
```

Comment:
Happy that the developer should be able to work it out from this, but worth noting that this won't actually work for me locally at least, unless I have set all of these environment variables beforehand. Using the local `.env` file (e.g. `docker run --env-file .env my-test-image tests/endtoend`) might be the better local option